plugins/woocommerce/client/admin/client/analytics/components/report-table/index.js

```diff
@@ -583,7 +583,7 @@ export default compose(
 		} = props;
 
 		/* eslint @wordpress/no-unused-vars-before-return: "off" */
-		const reportStoreSelector = select( REPORTS_STORE_NAME );
+		const reportStoreSelector = select( reportsStore );
```

Comment:
Although moving this line closer to where its use won't completely be able to eliminate the eslint off annotation, it might still be worth at least moving it first.